### DISCUSSION TITLE
Spring CP2K Developers Meeting in May

### DISCUSSION THREAD

**Discussion by robertschade** [cp2k/cp2k#2610] [Team Posts]
Dear @cp2k/cp2k-developers ,
We would like to plan the next CP2K developers meeting in May. To find a suitable date we would be happy if you could enter your availability in [https://terminplaner.dfn.de/9wkOxiIv7cSJLMDy](https://terminplaner.dfn.de/9wkOxiIv7cSJLMDy) till the 15th of April.
If you would like to contribute a topic, please simply add it to the slides
[https://docs.google.com/presentation/d/1ADc88hdXPUckNX-NhyXpgjzpzXGM68sbdGQdrQvOqzc/edit?usp=sharing](https://docs.google.com/presentation/d/1ADc88hdXPUckNX-NhyXpgjzpzXGM68sbdGQdrQvOqzc/edit?usp=sharing)
Best Wishes

**Comment by hfp** [comment]
Hi Robert, when visiting the Terminplaner, I am getting a webpage that only says "reset". Can you take a look?

**Comment by robertschade** [comment]
It works for me now. Maybe a temporary problem? Could you please try again?

**Comment by hfp** [comment]
No it does not work for me. Still showing the "reset". I tried outside of any company network, etc. but cannot get it to show the calendar. Has anyone else contributed to the poll successfully?

**Comment by robertschade** [comment]
Yes, already about 10 people have entered their availability. Does the main page [https://terminplaner.dfn.de/](https://terminplaner.dfn.de/) work for you?

**Comment by hfp** [comment]
Thank you, Robert!  I managed to vote (site was blacklisted on my end due to subdomain/ssl certificate or whatever).

Is the date already set?

**Comment by robertschade** [comment]
Ok, interesting. Our plan is to fix it by the end of this week.

**Comment by robertschade** [comment]
Dear @cp2k/cp2k-developers,
May 5th 14-16:00 CEST has turned out to be the best option. Thus, we would like to invite all of you to the next CP2K developers meeting. Please also forward this invitation to other people that might be interested.
If you would like to contribute a topic or question to discuss, please simply add it to the slides at
https://docs.google.com/presentation/d/1ADc88hdXPUckNX-NhyXpgjzpzXGM68sbdGQdrQvOqzc/edit?usp=sharing

Video conference information:
https://uni-paderborn-de.zoom.us/j/95035325826?pwd=UXhFMXlNYTJBS1VnSHZtYjFJblNXZz09
Meeting ID: 950 3532 5826
Passcode: 543520

Best Wishes
Robert Schade

**Comment by rzk1** [comment]
Hi Sylvester,

Here is the CP2K meeting info.

Rustam

**Comment by robertschade** [comment]
Dear @cp2k/cp2k-developers,
I have uploaded the slides to [https://www.cp2k.org/dev:meetings](https://www.cp2k.org/dev:meetings). Please let me know if you have material to add or if I should change anything.
We are planning to organize the in-depth discussion meeting on GPU acceleration in June. We will send around a scheduling survey end of May.
Best Wishes
Robert Schade
